ElementX/Sources/Services/Room/RoomPermissions.swift

```diff
@@ -45,46 +50,75 @@ struct RoomPermissionsSetting: Identifiable {
         default: fatalError("Unexpected key path: \(keyPath)")
         }
     }
+    
+    /// Can the setting be edited
+    var isDisabled: Bool {
+        switch ownPowerLevel {
+        case .value(let ownValue):
+            ownValue < value
+        case .infinite:
+            false
+        }
+    }
+    
+    /// All of the available roles that this setting can be configured with.
```

Comment:
Minor but maybe we should have this here and the original back on the `static let`?
```suggestion
    /// All of the available roles that this setting is allowed to be configured with.
```